tools/pom.xml

```diff
@@ -8,7 +8,7 @@
 	<modelVersion>4.0.0</modelVersion>
 
 	<artifactId>aijar-tools</artifactId>
-  <version>1.0.17-SNAPSHOT</version>
+  <version>2.0.0-SNAPSHOT</version>
```

Comment:
@dkayiwa Do try this with 2.0.5-SNAPSHOT with the released version does not work at mine